### ISSUE TITLE
Rake task rake scholarsphere:solr:compare is broken

### ISSUE THREAD

**Issue by hectorcorrea** [psu-libraries/scholarsphere-3#161]
This task still references Rubydora. It needs to be updated to work with Fedora 4.

This is the error that I get:

```
$ rake scholarsphere:solr:compare
/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/sufia-models-6.0.0/lib/tasks/sufia-models_tasks.rake:13: warning: already initialized constant FULLTEXT_JARS
/opt/heracles/deploy/scholarsphere/shared/bundle/ruby/2.1.0/gems/sufia-models-6.0.0/lib/tasks/sufia-models_tasks.rake:13: warning: previous definition of FULLTEXT_JARS was here
DEPRECATION WARNING: Blacklight.solr is deprecated and will be removed in 6.0.0. Use Blacklight.default_index.connection instead. (called from block (3 levels) in <top (required)> at /opt/heracles/deploy/scholarsphere/releases/20150428185938/lib/tasks/scholarsphere-solr.rake:31)
rake aborted!
NoMethodError: undefined method `connection_for_pid' for ActiveFedora::Base:Class
/opt/heracles/deploy/scholarsphere/releases/20150428185938/lib/tasks/scholarsphere-solr.rake:60:in `initialize_rubydora_connection'
/opt/heracles/deploy/scholarsphere/releases/20150428185938/lib/tasks/scholarsphere-solr.rake:33:in `block (3 levels) in <top (required)>'
Tasks: TOP => scholarsphere:solr:compare
(See full trace by running task with --trace)
```

**Comment by awead** [comment]
@hectorcorrea This was raised in #137 and fixed in #143 but was merged in to the development branch and not 2.2.3. We could cherry-pick it for 2.2.4 or just cut a new release off of develop. @cam156 thoughts?

**Comment by hectorcorrea** [comment]
Perfect. I wasn't aware that this had been reported and fixed before. I just wanted not to lose track of it but if we already have it in development then I'll close this ticket.
